src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -209,9 +211,11 @@ class Toolbar extends Component {
    * @returns {undefined}
    */
   UNSAFE_componentWillReceiveProps(nextProps) {
-    if (nextProps.pathname !== this.props.pathname) {
+    if (nextProps.pathname !== this.state.currentPathname) {
```

Comment:
@MdSahil-oss are you sure we need to keep the state for this? I can see the pathname change in the contents view. So the route should be updating, if it's not, then it should in the parent component or how this component gets the pathname prop.

Reply:
@sneridagh Worked on this issue a long time ago, give me a while then I get back to you.

Reply:
@sneridagh I removed the state from the code and made changes optimal as per plone codebase. Now I'm sure its good to go.